src/main/java/io/github/bedwarsrel/BedwarsRel/Updater/ConfigUpdater.java

```diff
@@ -272,6 +272,10 @@ public void addConfigs() {
     Main.getInstance().getConfig().addDefault("specials.arrow-blocker.using-wait-time", 30);
     Main.getInstance().getConfig().addDefault("specials.arrow-blocker.item", "ender_eye");
     // </1.3.4>
+    
+    // <1.3.5>
+    Main.getInstance().getConfig().addDefault("spawnRessourcesInChest", true);
```

Comment:
We've always been using dash notation instead of camel-case in the config before. Should be changed.